themes/V3/5ePHB/style.less

```diff
@@ -795,6 +795,16 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.page:has(.partCover), 
```

Comment:
These have the `--ToC` tag, but it doesn't seem to properly exclude them; they still appear in the table of contents. I suspect this is because the tag is being applied to the `.page`, but the table of contents generator only looks for the tag on the header elements.

Reply:
A reminder here for future readers that CSS is used here in a funky way -- instead of applying in realtime to control styling of elements, it is instead first applied  during authoring, then the toc-generation snippet is run, which produces the toc brewtext. Further, the toc-generation snippet isn't looking at style rules directly applied to heading elements, it is checking to see if they are affected by them (by virtue of the cascade of CSS). This is done via `getComputedStyle(heading).getPropertyValue('--TOC');`

(btw, this also means the CSS can be altered after TOC generation and it won't be applied to the previously generated TOC — the current styling CSS could appear to be lying.)

That said .. if the emperical test of running the snippet is that the exclusions are not being applied .. then something is indeed broken.

Reply:
From what I can see, the CSS is not being inherited down to children elements.

Reply:
I see otherwise .. right-click the heading in the console and choose Store as Global Variable, and then in console do `getComputedStyle(temp1).getPropertyValue('--TOC')`.

The property is computed.

Reply:
The reason the toc-snippet is failing to exclude is because line 38 has `if(ToCExclude != '"exclude"') {` .. which is testing the `getComputedStyle` with `"exclude"` .. but the stylesheet is using `exclude` (no quotes).